server/resp/src/main/java/org/infinispan/server/resp/RespServer.java

```diff
@@ -51,13 +65,14 @@ protected void startInternal() {
             }
             builder.encoding().key().mediaType(MediaType.APPLICATION_OCTET_STREAM_TYPE);
```

Comment:
Wouldn't it be better, as you suggested a while back, to default to protobuf, so that these caches can be interoperable ?

Reply:
lets try this one

Reply:
How would that work with RESP though? I wouldn't expect the user to be supplying byte[] in the protobuf format for its key and values.